action.yml

```diff
@@ -60,3 +61,10 @@ runs:
       shell: bash
       run: opensafely log-stats
       working-directory: ${{ inputs.directory }}
+    - name: Build honeycomb
```

Comment:
Building the docker image as part of the action run feels a bit weird to me. Don't we want to be building it elsewhere, pushing to ghcr.io, and then just running it here?

Reply:
Yes, I wondered if we should be.  I haven't pushed an image yet though, so it'd have nothing to run.  If it did build, it does the cache_from thing, so it'd be quick.  I also wondered if the dockerfile should be somewhere else, in its own repo, or with python-docker, rather than here
I can push up an image and update this step - it shouldn't change often